examples/mobile/UIComponents/components/controls/radio.js

```diff
@@ -4,7 +4,7 @@
 	 * radios - NodeList object for radios
 	 * radioresult - Indicator for active radio
 	 */
-	var page = document.getElementById("radio-demo"),
+	var page = document.querySelector(".ui-page"),
```

Comment:
In case if app has started from index.html page (main UI Components app page) there are a few containers with .ui-page class. We cannot use this selector here.

Reply:
Ok. But in this situation (radio.js file) isn't it launched only for radio.html file?

Reply:
Yes, radio.js file used only with this page, but TAU only appends active page to document. Page is removed only in case if is loaded from external document and it is removed from history stack.

![image](https://user-images.githubusercontent.com/38784474/61111800-ae67ef80-a48a-11e9-8dca-1059102af4ea.png)